pkg/apis/core/types.go

```diff
@@ -1442,6 +1442,12 @@ type LocalVolumeSource struct {
 	// Block devices can be represented only by VolumeMode=Block, which also requires the
 	// BlockVolume alpha feature gate to be enabled.
 	Path string
+
+	// Filesystem type to mount.
+	// Must be a filesystem type supported by the host operating system.
+	// Ex. "ext4", "xfs", "ntfs". Implicitly inferred to be "ext4" if unspecified.
+	// +optional
+	FSType string
```

Comment:
should be a pointer, as the field is optional

Reply:
Other persistent volume sources are just using string, see https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/core/types.go#L578
https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/core/types.go#L960
...

Reply:
New optional fields to existing types need to be pointers for backwards compatibility.  See https://github.com/kubernetes/kubernetes/issues/50121

Reply:
ah sorry, I think we may not be able to default the field, because it would not work on windows.